### DISCUSSION TITLE
Slack Authentication Provider is not visible for login

### DISCUSSION THREAD

**Discussion by tboclothing** [outline/outline#3346] [Self Hosting] [answered]
I'm new to Outline and I want to self host Outline on our server. I followed the [official instructions](https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/slack-sgMujR8J9J) and able to setup everything. 

For the authentication , We are using Google and Slack. Google login is working fine. But **Login with Slack** option is not coming on home page. 

I checked in code and found that in `server/routes/api/auth.ts` file in `team.authenticationProviders` array we don't have Slack identifier.

Could you please help in this. 
Thanks!

**Comment by Limezy** [comment, marked as answer]
As per now, only one authentification method is supported (+ emails). So if your first user has used google, then the next won't have the choice. 
It has been discussed already and Tom has started working on it. You can have a look at the PR list for reference.

**Reply by almereyda** [reply]
See #2986 for details.
